exchanges/order/order_test.go

```diff
@@ -652,41 +545,30 @@ func TestFilterOrdersByPairs(t *testing.T) {
 		currency.NewPair(currency.DOGE, currency.RUB),
 	}
 	FilterOrdersByPairs(&orders, currencies)
-	if len(orders) != 4 {
-		t.Errorf("Orders failed to be filtered. Expected %v, received %v", 3, len(orders))
-	}
+	assert.Lenf(t, orders, 4, "Orders failed to be filtered. Expected %v, received %v", 3, len(orders))
 
 	currencies = []currency.Pair{
 		currency.NewBTCUSD(),
 		currency.NewPair(currency.LTC, currency.EUR),
 	}
 	FilterOrdersByPairs(&orders, currencies)
-	if len(orders) != 3 {
-		t.Errorf("Orders failed to be filtered. Expected %v, received %v", 2, len(orders))
-	}
+	assert.Lenf(t, orders, 3, "Orders failed to be filtered. Expected %v, received %v", 2, len(orders))
```

Comment:
Same nit as before but also we're expecting 3 but also mentioning 2, will stop reporting these from now on 😂